lib/liquid/translators/tags/custom_tag.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Liquid.Translators.Tags.CustomTag do
+  alias Liquid.{Template, Tag}
+
+  def translate(custom_name: name, custom_markup: markup) do
```

Comment:
Can you add @doc for public functions and @specs for all functions?

Reply:
Done.